cfg.sample.toml

```diff
@@ -44,6 +44,13 @@ try_users = []
 # Auto-squash commits. Requires the local Git command
 #autosquash = true
 
+# Whether homu should push a temporary merge commit that causes
```

Comment:
To be clear, this only applies if `linear=true`, right? If so, that should be mentioned here.